superset/views/database/forms.py

```diff
@@ -402,3 +417,130 @@ def at_least_one_schema_is_allowed(database: Database) -> bool:
             'Use [""] for empty string.'
         ),
     )
+
+
+class ColumnarToDatabaseForm(DynamicForm):
+    # pylint: disable=E0211
+    def columnar_allowed_dbs() -> List[Database]:  # type: ignore
```

Comment:
I wonder why this wasn't `@staticmethod`?? Can we update that here and in the other classes

Reply:
Adding that decorator will throw `TypeError: 'staticmethod' object is not callable`